doc-Monitoring_Alerts_and_Reporting/topics/Integrating_Prometheus_Alerts.adoc

```diff
@@ -0,0 +1,130 @@
+====
+
+. Navigate to menu:Control[Explorer], then click *Alert Profiles* in the accordion menu.
+. Click to expand *Node Alert Profiles*, then click *Prometheus Node Profile*. 
+. Click image:1847.png[Configuration] (*Configuration*), then click image:1851.png[] (*Edit assignments for this Alert Profile*). Assign the profile to *The Enterprise* from the *Assign To* list.
+. Click *Save*.
+. Click to expand *Provider Alert Profiles*, then click *Prometheus Provider Profile*. 
+. Click *Configuration*, then click *Edit assignments for this Alert Profile*. Assign the profile to *The Enterprise* from the *Assign To* list.
+. Click *Save*.
++
+[NOTE]
+====
+Alerts triggered before assigning the profiles to the enterprise will not appear in {product-title_short} at all.
+====
++
+
+
+==== Adding OpenShift Containers Providers and Prometheus Alert Endpoints
+
+Complete the following procedure using the {product-title_short} user interface to add OpenShift Containers Provider and Prometheus alerts endpoints.
+
+. Navigate to menu:Compute[Containers > Providers].
+. Click  image:1847.png[Configuration] (*Configuration*), then click  image:1862.png[Add New Containers Provider] (*Add New Containers Provider*).
+. Enter a *Name* for the provider.
+. From the *Type* list, select *OpenShift Container Platform*.
+. Enter the appropriate *Zone* for the provider. If you do not specify a zone, it is set to `default`.
+. From the *Alerts* list, select *Prometheus* to enable external alerts. Selecting *Prometheus* adds an *Alerts* tab to the lower pane to configure the Prometheus service. Alerts are disabled by default.
+. In the *Default* tab, add the default endpoints for the OpenShift provider: 
+.. Select a *Security Protocol* method to specify how to authenticate the provider:
+* *SSL*: Authenticate the provider securely using a trusted Certificate Authority. Select this option if the provider has a valid SSL certificate and it is signed by a trusted Certificate Authority. No further configuration is required for this option.
+* *SSL trusting custom CA*: Authenticate the provider with a self-signed certificate. For this option, copy your provider’s CA certificate to the *Trusted CA Certificates* box in PEM format.
++
+[NOTE]
+====
+To obtain your OpenShift Container Platform provider's CA certificate, run the `oc get secret` command on your provider, substituting values for your provider and token as needed. To obtain a token for your provider, see link:https://access.redhat.com/documentation/en-us/red_hat_cloudforms/4.6-beta/html-single/managing_providers/#Obtaining_OpenShift_Container_Platform_Management_Token[Obtaining an OpenShift Container Platform Management Token].
+
+For example:
+
+  # oc get secret --namespace management-infra management-admin-token-8ixxs --template='{{index .data "ca.crt"}}' | base64 --decode
+  
+Paste the output (a block of text starting with `-----BEGIN CERTIFICATE-----`) into the *Trusted CA Certificates* field.
```

Comment:
there is a mixup between the ca certificate which is the main openshift CA available in `/etc/origin/master/ca.crt` and the management-admin service account. 

I don't this this section should exist here only what we have in the endpoints sections, for example: https://github.com/ManageIQ/manageiq_docs/blob/6ef2bef2d9cfddbceb5e1617ccacae626cb15fe5/doc-Managing_Providers/topics/Editing_a_Containers_Provider.adoc
I see that page needs some changes as well... not sure if related to this work but we no longer have a "hawkular tab" but "Metrics" and "Alerts" tabs...

Reply:
"there is a mixup between the ca certificate which is the main openshift CA available in /etc/origin/master/ca.crt and the management-admin service account."
- could you clarify this more please? so which of the two methods to obtain the CA certificate should be covered here?

"I don't this this section should exist here only what we have in the endpoints sections, for example: https://github.com/ManageIQ/manageiq_docs/blob/6ef2bef2d9cfddbceb5e1617ccacae626cb15fe5/doc-Managing_Providers/topics/Editing_a_Containers_Provider.adoc"
-Continuing from the above comment, so you mean the current section is not needed here as the user needs to only edit an existing containers provider to include the step on alert endpoint (alert tab) as part of configuring and enabling Prometheus alerts in CloudForms? so edit provider and add this:
----
Add the Prometheus alerts endpoints in the Alerts tab:

a.Select a Security Protocol method to specify how to authenticate the service:

SSL: Authenticate the provider securely using a trusted Certificate Authority. Select this option if the provider has a valid SSL certificate and it is signed by a trusted Certificate Authority. No further configuration is required for this option.
SSL trusting custom CA: Authenticate the provider with a self-signed certificate. For this option, copy your provider’s CA certificate to the Trusted CA Certificates box in PEM format.
SSL without validation: Authenticate the provider insecurely using SSL. (Not recommended)
b. Enter the Hostname or IPv4 or IPv6 address of the provider.
c. Enter the API Port if your Prometheus provider uses a non-standard port for access. The default port is 443.
d. Click Validate to confirm that CloudForms can connect to the alerts service.
Click Add.
----
I was considering making this as an end-to-end example which includes all of the steps. 

"I see that page needs some changes as well... not sure if related to this work but we no longer have a "hawkular tab" but "Metrics" and "Alerts" tabs..." 
-Ack, will create a separate bug to track / update that section, also since hawkular no longer exists. thanks.

Reply:
> could you clarify this more please? so which of the two methods to obtain the CA certificate should be covered here?

There are two separate things that two separate places in the UI
"Trusted CA Certificates" is relevant to all endpoints (default, metrics, alerts) they can all be obtained from the `/etc/origin/master/ca.crt`

"Token" Is what you get from `oc get secret` command, thats relevant only for the default endpoint.

Reply:
@moolitayer Thanks for the clarification. I've slightly modified the two steps to avoid any mix up. Hope it makes sense.